lib/cmd/fh3/admin/domains/check.js

```diff
@@ -23,8 +23,10 @@ module.exports = {
     return "/box/api/domains/check?domain=" + domain;
   },
   'method' : 'get',
-  'postCmd': function(params, cb) {
-    params._table = common.createObjectTable(params);
+  'postCmd': function(argv, params, cb) {
+    if (!argv.json) {
+      params._table = common.createObjectTable(params);
+    }
     return cb(null, params);
```

Comment:
Hi @wtrocki, I am not against your suggestion. I just think we should keep the approach to do the things step by step. I think we can follow doing in a simple way and after in the end of the epic we can create a task just to check it and follow the best way to implement/improve .. just step by step for we have the visibility and take the best path.